examples/examples_go_test.go

```diff
@@ -370,6 +393,35 @@ func (st labelsState) validateTransitionTo(t *testing.T, st2 labelsState) {
 	integration.ProgramTest(t, &opts)
 }
 
+func (st labelsState) expectedLabelsPRC(prev labelsState) map[string]string {
+	// Note that the upstream provider actually takes a "" value for a label to mean "keep the previous value".
```

Comment:
This is highly suspect, do you have a code reference? This reproduces in TF? I can think of strong customer feedback in AWS that empty tag values should not be specially treated, suspect the same may be the case here.

Reply:
I see this is present in TF.  Still in AWS we were asked to fix it even though TF had this problem. The PR looks good to me but I'd request tracking this problem in the issue tracker so that we can at least call this out to users appropriately when PRC is rolling out.

Reply:
I'll open an issue and link it in the rollout issue, thanks